tests/authoring/Applicability/ApplicableToComponent.fs

```diff
@@ -479,11 +483,14 @@ stack: unavailable
     let ``renders unavailable`` () =
         markdown |> convertsToHtml """
 <p class="applies applies-block">
-	<span class="applicable-info" data-tippy-content="Not available on Elastic&nbsp;Stack unless otherwise specified.">
+	<span class="applicable-info" data-tippy-content="Unavailable since 8.0.">
```

Comment:
Unavailable should never show a version - we may have missed this case in the specs; now that I think of it

If something is no longer in the product, we'd use "removed 8.0", with the removed lifecycle.
Unavailable strictly means that this is simply not possible to do with that product